src/clj/rems/css/styles.clj

```diff
@@ -781,9 +781,7 @@
                     :font-size (u/rem 1.5)
                     :line-height 1.1
                     :font-family (get-theme-attribute :font-family)
-                    :color (get-theme-attribute :collapse-color)}
-     ;; TODO figure this out
-     [:&.alert-danger {:color (get-theme-attribute :danger-color "inherit")}]]]
```

Comment:
Does this make it impossible to customize this or was this simply not needed to change the alert colors?

Reply:
Customization is still possible via the `:text-danger` and `:bg-danger` vars. It seems to me `:danger-color` was a mistake or a leftover from some previous stuff.